docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+# v2 syntax
+version: '2'
+
+services:
+  # The database service
+  postgres:
+    image: postgres:9.6
+    container_name: jarbas-postgres
+    environment:
```

Comment:
What is a good practice when it comes to sensible info like this? Does Docker reads from server envvar or something?